forem/forem
# Added dropdown to display ads / billboards

## What type of PR is this? (check all applicable)


## Description

This PR adds the org name and hardcoded dropdown menu to all currently existing billboards.


## Related Tickets & Documents



- Closes https://github.com/forem/forem-internal-eng/issues/531

## QA Instructions, Screenshots, Recordings

**Reference Video**: https://www.loom.com/share/92859e41b05745b99423a653d83cc5f9

**Figma Designs**: https://www.figma.com/file/9UfSjNYJs745w3Q8JCUqsL/Small-Wins---%E2%80%A8Core-Experience-%E2%9A%A1%EF%B8%8F?node-id=1157%3A5067&t=d2xqTG6IUYL6oZq9-0

**How to test?**
1. Sign in as admin locally.
2. Create a `billboard` page on http://localhost:3000/admin/customization/pages
3. Make sure that the value of `slug` is `billboards` and select `Is top level path?`
4. Once finished, visit http://localhost:3000/billboard to see that the page is visible, if not, there was some mistake in creating the page.
5. Now visit http://localhost:3000/admin/customization/display_ads to create different types of ads (5 types of ads a possible) and check them on either home page or on article page.
6. Once you visit any billboard/display ad, click on three-dots/overflow-menu icon to see the dropdown menu.
7. Click on the two available options of dropdown menu to see if they work correctly. 

### Output
<img width="269" alt="Screenshot 2023-02-24 at 2 23 36 PM" src="https://user-images.githubusercontent.com/9396084/221135199-fdedf6fa-9f21-4797-85f3-96d51ddd1d0c.png"> <img width="282" alt="Screenshot 2023-02-24 at 2 24 05 PM" src="https://user-images.githubusercontent.com/9396084/221135208-c03d5d41-f8ed-479f-8328-e0e0ae694291.png">
<img width="300" alt="Screenshot 2023-02-24 at 2 23 41 PM" src="https://user-images.githubusercontent.com/9396084/221135220-50541ea6-d0e9-41db-a49b-3e5ad71515c0.png"> <img width="311" alt="Screenshot 2023-02-24 at 2 24 11 PM" src="https://user-images.githubusercontent.com/9396084/221135226-639ff7af-c7d6-41f5-9176-e8af808371a9.png">

Latest UI Output: https://www.loom.com/share/7c80770fd75b4be79e531c8731d52f7e

<img width="374" alt="Screenshot 2023-02-24 at 2 08 04 PM" src="https://user-images.githubusercontent.com/9396084/221132027-3f289ef4-0ae5-4f5c-b28c-ec6e7dbed207.png">
<img width="290" alt="Screenshot 2023-02-24 at 2 25 40 PM" src="https://user-images.githubusercontent.com/9396084/221135486-aae9abca-a416-4f55-8ac8-d3be06fb5f52.png">
<img width="281" alt="Screenshot 2023-02-24 at 2 08 29 PM" src="https://user-images.githubusercontent.com/9396084/221132055-ace974d6-33fd-4f40-a364-eedde792da8b.png">
<img width="430" alt="Screenshot 2023-02-24 at 2 08 37 PM" src="https://user-images.githubusercontent.com/9396084/221132060-619ebe9d-f285-45cf-96bf-a448d4c6f7f2.png">
<img width="950" alt="Screenshot 2023-02-24 at 2 08 46 PM" src="https://user-images.githubusercontent.com/9396084/221132063-54047266-5eef-44a0-a267-82d22d8c4700.png">


### UI accessibility concerns?

Improves accessibility by making users understand what a billboard is and access to settings.


## Added/updated tests?

- [x] No tests added https://github.com/forem/forem/pull/19106#pullrequestreview-1301412835

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._


## [optional] Are there any post deployment tasks we need to perform?

@maestromac Can you deploy this on canary1 or canary2 for testing?
